core/tools.js

```diff
@@ -1463,6 +1463,36 @@
 		},
 
 		/**
+		 * Detects, which mouse button generated given DOM event
+		 *
+		 * @since 4.7.3
+		 * @param {CKEDITOR.dom.event} evt DOM event.
+		 * @returns {Number}
```

Comment:
You made a lie here! 🙀 It can also return a boolean. That has to be documented. 😉

Reply:
Yup, I'm a liar :( But I've fixed it.